spec/services/merge/merge_organisations_service_spec.rb

```diff
@@ -221,8 +221,8 @@
               expect(absorbed_scheme.locations.count).to eq(1)
               absorbed_location = absorbed_scheme.locations.first
 
-              expect(absorbed_scheme.startdate).to eq(Time.zone.today + 1.month)
```

Comment:
be explicit here now we're defaulting to TimeWithZone - we're doing this elsewhere in this file but it wasn't relevant for these fields as they had unspecified precision before